lib/route_translator/translator.rb

```diff
@@ -6,8 +6,6 @@
 module RouteTranslator
   module Translator
     class << self
```

Comment:
Improvement to `add_route`:
https://github.com/enriclluelles/route_translator/commit/5ad6bf13f7dc82008afecc91ae08fcec21405ec3